views/dataset/suggest_projects.templ

```diff
@@ -35,30 +35,34 @@ templ SuggestProjects(c *ctx.Ctx,  dataset *models.Dataset, hits []*models.Proje
 													EU Funding programme: Horizon 2020
 												</div>
 											}
-											if hit.StartDate != "" && hit.EndDate !="" {
+											if hit.StartDate != "" && hit.EndDate != "" {
 												<div class="pe-4 me-4 border-right">
 													Period: { hit.StartDate } – { hit.EndDate }
 												</div>
 											}
 											<div class="me-43">
-												<a class="c-link c-link-muted" target="blank">Read more <i class="if if-external-link if--small"></i></a>
+												<a
+													class="c-link c-link-muted"
+													href={ templ.URL("https://biblio.ugent.be/project/" + hit.ID) }
```

Comment:
[c.FrontendURL](https://github.com/ugent-library/biblio-backoffice/blob/dev/ctx/ctx.go#L115) is also available

Reply:
Thanks. Now using this in combination with the URLBuilder.